packages/charts/react-charting/src/components/Legends/Legends.base.tsx

```diff
@@ -195,16 +195,6 @@ export class LegendsBase extends React.Component<ILegendsProps, ILegendState> {
     return { primary, overflow };
   };
 
-  /**
-   * Determine whether the component is in "controlled" mode for selections, where the selected legend(s) are
-   * determined entirely by props passed in from the parent component.
-   */
-  private _isInControlledMode = (): boolean => {
-    return this.props.canSelectMultipleLegends
-      ? this.props.selectedLegends !== undefined
-      : this.props.selectedLegend !== undefined;
-  };
-
```

Comment:
This is enforcing that the updated legend props are passed back by the caller to the legends control. 
Without this the user selected legend is getting lost.
We cannot enforce this behavior to the caller.
We would need to add a boolean flag to enforce strict controlling of legends. This will be taken up later if needed.